test/spmd/test_inference_spmd_dynamo_imagenet.py

```diff
@@ -0,0 +1,238 @@
+import args_parse
```

Comment:
We also need to basic test similar to https://github.com/pytorch/xla/blob/master/test/dynamo/test_dynamo.py#L45 to check the correctness of the dynamo + SPMD. In this case I guess you should expect `dynamo + SPMD` yield the same result as `SPMD` along to test this integration. Test can be either under `SPMD` dir or `dynamo` dir. Currently we don't even know if SPMD is emitting the correct result when used with dynamo(I suppose we did some e2e test).

Reply:
Tried adding a test, currently crashing on `res_xla_spmd_dynamo = self.fn_simple_dynamo(sharded_x, sharded_y)`

Reply:
let me know if you need help.